### PR TITLE
Add Technical Gadgets pages

### PR DESCRIPTION
**Pull request by svr333**
## Changes
- Adds Technical Gadgets page
- Adds Jetpacks page
- Adds Multi Tools page
- Adds Jetboots page
- Fixes multimeter links
- Adds Android-Memory-Core to Technical Components page

### CODE REVIEW

**Review by Phoenix-Starlight — APPROVED** [review]

File: pages/Multi-Tools.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+Multi Tools are part of the [Technical Gadgets](https://github.com/Slimefun/Slimefun4/wiki/Technical-Gadgets) category. They need to be charged in a [Charging Bench](https://github.com/Slimefun/Slimefun4/wiki/Charging-Bench) before usage.
+
+To use, right click, to switch modes, crouch and right click.
```
Comment (WalshyDev) [inline pages/Multi-Tools.md:3]:
```suggestion
Right-click to use. Crouch + right-click to switch modes\
```

File: pages/Multi-Tools.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+Multi Tools are part of the [Technical Gadgets](https://github.com/Slimefun/Slimefun4/wiki/Technical-Gadgets) category. They need to be charged in a [Charging Bench](https://github.com/Slimefun/Slimefun4/wiki/Charging-Bench) before usage.
+
+To use, right click, to switch modes, crouch and right click.  
+Each time you use it, it will use up some energy.
```
Comment (WalshyDev) [inline pages/Multi-Tools.md:4]: what's "some energy" how much? does it depend on the tier?

File: pages/Technical-Gadgets.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+* [Jet Boots - VI](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Jet Boots - VII](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Armored Jet Boots](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Multimeter](https://github.com/Slimefun/Slimefun4/wiki/Technical-Gadgets#multimeter)
+
+### Parachute
+The Parachute can be used to slowly descent when you're in the air. You need to wear it and crouch for it to activate.
+
+### Solar Helmet
+The Solar Helmet, when worn, will charge any items and armor in that user's inventory.
```
Comment (WalshyDev) [inline pages/Technical-Gadgets.md:36]:
```suggestion
The Solar Helmet when worn will charge any electric armor and anything electric in the player's hands.
```

**Review by WalshyDev — CHANGES_REQUESTED** [review]

**Review by WalshyDev — APPROVED** [review]
lgtm

File: pages/Technical-Gadgets.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+* [Multimeter](https://github.com/Slimefun/Slimefun4/wiki/Technical-Gadgets#multimeter)
+
+### Parachute
+The Parachute can be used to slowly descent when you're in the air. You need to wear it and crouch for it to activate.
+
+### Solar Helmet
+The Solar Helmet when worn will charge any electric armor and anything electric in the player's hands.
+
+### Night Vision Goggles
+When worn, this piece of head armour gives you night vision.
```
Comment (WalshyDev) [inline pages/Technical-Gadgets.md:39]: We should really standardize a language here. Do we want British English or American English?

File: pages/Technical-Gadgets.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+* [Jet Boots - VI](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Jet Boots - VII](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Armored Jet Boots](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Multimeter](https://github.com/Slimefun/Slimefun4/wiki/Technical-Gadgets#multimeter)
+
+### Parachute
+The Parachute can be used to slowly descent when you're in the air. You need to wear it and crouch for it to activate.
+
+### Solar Helmet
+The Solar Helmet when worn will charge any electric armor and anything electric in the player's hands.
```
Comment (WalshyDev) [inline pages/Technical-Gadgets.md:36]: You use American English here :NotLikeThis:

**Review by WalshyDev — CHANGES_REQUESTED** [review]

**Review by WalshyDev — DISMISSED** [review]
LGTM

File: pages/Jet-Boots.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+Jet Boots are part of the [Technical Gadgets](https://github.com/Slimefun/Slimefun4/wiki/Technical-Gadgets) category. They need to be charged in a [Charging Bench](https://github.com/Slimefun/Slimefun4/wiki/Charging-Bench) before usage.
+
+They can be used to fly around using electricity. Put the Jet Boots in your boots slot and crouch to use.
+
+## Jet Boots Tiers
+
+|  Tier   | Storage Capacity | Speed | Acurracy |
```
Comment (Sfiguz7) [inline pages/Jet-Boots.md]:
```suggestion
|  Tier   | Storage Capacity | Speed | Accuracy |
```

File: pages/Technical-Gadgets.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+* [Jet Boots - III](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Jet Boots - IV](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Jet Boots - V](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Jet Boots - VI](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Jet Boots - VII](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Armored Jet Boots](https://github.com/Slimefun/Slimefun4/wiki/Jet-Boots)
+* [Multimeter](https://github.com/Slimefun/Slimefun4/wiki/Technical-Gadgets#multimeter)
+
+### Parachute
+The Parachute can be used to slowly descent when you're in the air. You need to wear it and crouch for it to activate.
```
Comment (Sefiraat) [inline pages/Technical-Gadgets.md:33]:
```suggestion
The Parachute can be used to slowly decend while you're in the air. You need to wear it and crouch for it to activate.
```

**Review by Sfiguz7 — APPROVED** [review]
LGTM

File: pages/Jet-Boots.md (hunk 6 above)
Comment (Sfiguz7) [reply pages/Jet-Boots.md]: We were in a call and he already changed the requested change

**Review by Sefiraat — APPROVED** [review]
lgtm

**Pull request merged**